### PR TITLE
Upgrade to Sphinx 3.1

### PR DESCRIPTION
**Pull request by mxsasha**
- [x] Solve [rendering issue with lists](https://writethedocs-www--1255.org.readthedocs.build/conf/portland/2020/schedule/)
- [x] Video pages are broken

### CODE REVIEW

**Comment by plaindocs** [comment]
This is exciting too!

**Comment by mxsasha** [comment]
From what I've seen everything works fine now, please take a look at the read the docs build to see if you find any rendering issues.

**Comment by plaindocs** [comment]
* Videos :heavy_check_mark: 
* Conf :heavy_check_mark: 
* Random other pages :heavy_check_mark:

**Review by plaindocs — APPROVED** [review]
Without digging too deep into the 200 odd similar files changes, looks fine to me.

File: requirements.txt [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
+ablog==0.10.6
 tox==3.15.2
 sphinx_autobuild==0.7.1
 PyYAML>=4.2b1
 docutils==0.16
 csvkit==1.0.5
 yamllint==1.23.0
 pathlib==1.0.1
-git+https://github.com/ericholscher/datatemplates@add-key-and-context#egg=sphinxcontrib.datatemplates-dev
+sphinxcontrib.datatemplates==0.6.1
```
Comment (ericholscher) [inline requirements.txt:11]: Thanks for fixing this @mxsasha, it's been on my list for a while :)

**Review by ericholscher — APPROVED** [review]

**Comment by ericholscher** [comment]
This seems to break the links to the speaker abstract on the schedule: https://writethedocs-www--1255.org.readthedocs.build/conf/portland/2019/schedule/#monday-may-20 -- if you click on one of them, I believe because we aren't passing in the context variables to these.

**Pull request merged**